Sources/SwiftInspectorVisitors/NestableTypeVisitor.swift

```diff
@@ -159,14 +159,18 @@ public final class NestableTypeVisitor: SyntaxVisitor {
         genericRequirementVisitor.walk(genericWhereClause)
       }
 
+      let propertyVisitor = PropertySyntaxVisitor()
+      propertyVisitor.walk(node)
```

Comment:
could you add a `members: MemberDeclListSyntax { get }` property to `protocol NestableDeclSyntax`? I think all the nestable types have this, and that'll save us a teensy bit of parsing time by allowing us to walk just the `members` tree rather than the whole declaration.

Reply:
https://github.com/fdiaz/SwiftInspector/pull/100/commits/40bf7fc9632eecde178ca22be3a5016433050ec9